### PR TITLE
Implements stats for Charitoni Sling

### PR DESCRIPTION
**Pull request by Aeltorian**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds the missing stats on Charitoni Sling.

## Steps to test these changes

Put it on and notice the stats finally take effect.

### CODE REVIEW

File: scripts/globals/effects/rampart.lua [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-    if subPower > 0 then
-        target:delMod(xi.mod.FASTCAST, subPower)
-    end
-
-    for i = xi.mod.SLASH_SDT, xi.mod.DARK_SDT do
-        target:delMod(i, -power)
-    end
-end
-
-return effectObject
```
Comment (Xaver-DaRed) [inline scripts/globals/effects/rampart.lua]: Why did you delete this file, sir?

Comment (Aeltorian) [reply scripts/globals/effects/rampart.lua]: I don't know what's going on. I may have done something in error. I went to PR one thing and suddenly the entire history PR gets committed.

Comment (Xaver-DaRed) [reply scripts/globals/effects/rampart.lua]: Always remember to check base first before creating a branch

**Comment by Xaver-DaRed** [comment]
The modifiers look ok, but you need to do a rebase to remove those uneeded commits. from a different branch

**Comment by Aeltorian** [comment]
> The modifiers look ok, but you need to do a rebase to remove those uneeded commits. from a different branch

I would if it'd let me do it... it's acting wierd on my end. I'll have to figure it out.

**Review by Xaver-DaRed — APPROVED** [review]

**Comment by CatsEyeXI** [comment]
good job!

**Review by zach2good — APPROVED** [review]

**Review by zach2good — CHANGES_REQUESTED** [review]
This also has `Emico Mantle` in it, is that intended?

**Comment by Aeltorian** [comment]
> This also has `Emico Mantle` in it, is that intended?

That was from a prior PR a few days ago. Wasn't intended for this particular PR. 

Edit: we could add it to this PR and close the other PR which actually has a weird branch/base error,  since these two edits are of the same nature.

**Comment by CatsEyeXI** [comment]
@Aeltorian if you need help cleaning this up shoot me a DM

**Comment by Aeltorian** [comment]
> @Aeltorian if you need help cleaning this up shoot me a DM

I believe I found the way to update the file just now.

**Review by zach2good — APPROVED** [review]
